ui/src/widgets/ui-text-input/UITextInput.vue

```diff
@@ -1,14 +1,20 @@
 <template>
-    <v-text-field
-        v-if="type !== 'textarea'" v-model="value"
-        :disabled="!state.enabled" class="nrdb-ui-text-field"
-        :label="label" :type="type" :rules="validation" variant="outlined" hide-details="auto" @update:model-value="onChange" @keyup.enter="onEnter" @blur="onBlur"
-    />
-    <v-textarea
-        v-else
-        v-model="value" :disabled="!state.enabled" class="nrdb-ui-text-field"
-        :label="label" variant="outlined" hide-details="auto" @update:model-value="onChange" @blur="send"
-    />
+    <v-tooltip :text="tooltip" :disabled="tooltip.length === 0" location="bottom">
+        <template v-slot:activator="{ props }">
```

Comment:
```suggestion
    <v-tooltip :text="tooltip" :disabled="!tooltip?.length" location="bottom">
        <!-- eslint-disable-next-line vue/no-template-shadow -->
        <template v-slot:activator="{ props }">
```

Safer logic check for older `ui-text-input` nodes that would have no `tooltip` defined. Also added a `eslint-disable` as it's complaining about the use of `props` here (which is the correct usage)

Reply:
@chrblabla if you can merge/accept this suggestion, then it's good to go. I don't have permissions to push to your branch directly, so couldn't resolve it myself